pkg/migrate/migrate.go

```diff
@@ -12,12 +13,22 @@ const (
 
 var (
 	migratedRepos = sets.NewString(
-		"openshift/jenkins-openshift-login-plugin/master",
+		"openshift/jenkins-openshift-login-plugin/*",
```

Comment:
Why not pass the regexes via a flag rathen that hardcoding them into the code?

Reply:
This is used only in the migrating phase and will be removed very soon.